earth_enterprise/src/common/performancelogger.h

```diff
@@ -83,8 +90,13 @@ class BlockPerformanceLogger {
 //
 // If you use multiple performance logging statements in the same scope you
 // must give each one a unique name.
+#ifdef LOG_PERFORMANCE
 #define BEGIN_PERF_LOGGING(name, op, ...) \
   BlockPerformanceLogger<PerformanceLogger> name(op, __VA_ARGS__)
 #define END_PERF_LOGGING(name) name.end()
+#else
+#define BEGIN_PERF_LOGGING
```

Comment:
I get compile errors if I don't specify `LOG_PERFORMANCE`. They can be fixed by adding an argument list here and on the line below - i.e., `BEGIN_PERF_LOGGING(...)`.